docs/DeveloperGuide.adoc

```diff
@@ -356,14 +372,14 @@ This section highlights the implementation consideration of the program.
 ** Pros: We only revert changes that are hard to change back (the view can easily be re-modified as no data are lost).
 ** Cons: User might think that undo also applies when the list is modified (undoing filtering for example),  only to realize that it does not do that, after executing `undo`.
 * **Alternative 2:** Include all commands.
-** Pros: Program might be more intuitive for the user.
-** Cons: User has no way of skipping such commands if he or she just want to reset the state of the PTMan and not the view.
+** Pros: PTMan might be more intuitive for the user.
+** Cons: User has no way of skipping such commands if he or she just want to reset the state of PTMan and not the view.
 **Additional Info:** See our discussion  https://github.com/se-edu/addressbook-level4/issues/390#issuecomment-298936672[here].
 
 ===== Aspect: Data structure to support the undo/redo commands
 
 * **Alternative 1 (current choice):** Use separate stack for undo and redo
-** Pros: Easy to understand for new Computer Science student undergraduates to understand, who are likely to be  the new incoming developers of our project.
+** Pros: Easy to understand for new Computer Science student undergraduates to understand, who are likely to be the new incoming developers of our project.
```

Comment:
`Easy to understand for new Computer Science student undergraduates to understand` pls